commands/monitoring/pipeline/get.go

```diff
@@ -0,0 +1,62 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/ionos-cloud/ionosctl/v6/commands/monitoring/pipeline/completer"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table/jsonpaths"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/jsontabwriter"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func MonitoringFindByIdCmd() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
+		Namespace: "monitoring",
+		Resource:  "pipeline",
+		Verb:      "get",
+		Aliases:   []string{"g"},
+		ShortDesc: "Retrieve a pipeline",
+		Example:   "ionosctl monitoring pipeline get --pipeline-id GATEWAYID --location de/txl",
+		PreCmdRun: func(c *core.PreCommandConfig) error {
+			if err := core.CheckRequiredFlags(c.Command, c.NS, constants.FlagPipelineID); err != nil {
+				return fmt.Errorf("missing pipeline ID flag:\n %w", err)
+			}
+
+			return nil
+		},
+		CmdRun: func(c *core.CommandConfig) error {
+			monitoringId := viper.GetString(core.GetFlagName(c.NS, constants.FlagPipelineID))
+
+			r, _, err := client.Must().Monitoring.PipelinesApi.PipelinesFindById(context.Background(), monitoringId).Execute()
+			if err != nil {
+				return fmt.Errorf("executing the Pipeline FindById comand:\n %w", err)
```

Comment:
for errors you need to explain what went wrong, and no new lines. Extra help with debugging what went wrong is also helpful

```suggestion
				return fmt.Errorf("failed getting the pipeline with ID '%s': %w", monitoringId, err)
```

also just a nitpick , and you can leave this as-is, but `monitoringId` variable is confusing and could have a better name i.e. `pipelineId`